MIP102/MIP102.md

```diff
@@ -84,3 +89,13 @@ MIP102c3 is a Process MIP component that allows the removal of one or multiple A
 - **Other Requirements**: None.
 
 MIP102c3 subproposals must use the template located at **[MIP102c3-Subproposal-Template.md](https://github.com/makerdao/mips/blob/master/MIP102/MIP102c3-Subproposal-Template.md)**.
+
+### MIP102c4: Simultaneous Edit Resolution Process
+
+Where voters approve multiple MIP102c2 Subproposals that seek to edit the same component(s) of a MIP or Alignment Artifact within the same Governance Cycle, the following process must be followed. This process only applies to the specific components that are being simultaneously edited. Other amended components that do not have edits in simultaneous proposals may be merged, unless they are dependent on the component that does have simultaneous edits.
+
+If the Governance Facilitators can consolidate the proposals into a single edit, this should be the initial course of action. For example, two separate proposals may make non-conflicting changes to wording of a component that can simply be merged into a single edit. Or alternatively, two edits could add new non-conflicting components to a MIP or Alignment Artifact, that can both be merged with a simple change to the numbering of the components. Any action taken by the Governance Facilitators in this regard should be documented publicly and preferably be available prior to the end date of the relevant ratification polls.
+
+In the event that the Governance Facilitators cannot rationalize the proposals into the single edit, it falls to voters to decide which of the conflicting subproposals they wish to accept. If there are two conflicting subproposals, a binary vote is appropriate. In the event there are more than two conflicting subproposals, the Governance Facilitators may choose a suitable polling method that allows voters to choose between multiple options. The difference between the two amendments must be clearly presented so that voters can make an informed choice on their preferred option.
```

Comment:
```suggestion
- Where simultaneous edits are _conflicting, logically incompatible_ and cannot be coherently reconciled, it falls to voters to decide which of the conflicting subproposals they wish to accept. If there are two conflicting subproposals, a subsequent binary vote is appropriate. In the event there are more than two conflicting subproposals, the Governance Facilitators may choose a suitable polling method that allows voters to choose between multiple options. The difference between the two amendments must be clearly presented so that voters can make an informed choice on their preferred option.
```